docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -0,0 +1,195 @@
+- `spec.github.allow` is used to set rules for what GitHub Actions will be able to
+authenticate by using the token. In this example, the `gravitational/example`
+repository is used, and this example repository should be replaced with your own repo.
+
+Most of the configuration settings used in the example configuration yaml `tokenconfig.yaml`
+do not have any variable settings, however the `allow` block or `spec.github.allow` has additional
+configuration options to more strictly define the scope of access granted to the token:
+
+| **Setting** | **Behavior** | **Example** |
+| ----------- | ----------- | ----------- |
+| `sub` | Defines a subject claim. | `repo:example-org/example-repo:environment:dev` |
+| `repository` | The repository where the actions will run. | `gravitational/example` |
+| `repository_owner` | The name of the organization that owns the repository where the actions will run.  | `example-org` |
+| `workflow`| The name of the workflow. | `example-workflow` |
+| `environment`| The name of the environment where the actions will run. | `dev` |
+| `actor` | The account that will perform the actions. | `username` |
+| `ref` | The git "ref" or reference that triggers the action. | `refs/heads/example` |
+| `ref_type` | The type of "ref" or reference that triggers the action. | `branch` |
+
+Once the configuration YAML file has been created, apply it to your cluster using `tctl`:
+
+```code
+$ tctl create tokenconfig.yaml
+```
+
+The token `github-demo` can now be observed with the following command, ready to be used by
+GitHub:
+
+```code
+$ tctl tokens ls
+```
+
+## Step 2/3. Creating your Machine ID bot
+
+With the join token for GitHub Actions created, the next step is to create the Machine ID bot,
+and ensure that is configured to use the created token. This bot can be created using a Teleport
+`configuration.yaml` file, however for the purposes of this guide you will create a simple configuration
+using `tctl` with any required configuration flags.
+
+The Machine ID bot created in this example will be used to access a specific node on the cluster
+via `tsh ssh`, and will therefore require a role that can access the cluster as needed. This example
+configuration will apply the `access` role, however care should be taken to either create or apply
+a role according to the principle of least privelege in production environments. Additionally, it
+should have explicit access to the cluster using a username created specifically for the bot user alone.
+
+Enter the following command from your host with `tsh` access, replacing the `username` value with
+the username affiliated with a node the bot will be required to access over a `tsh ssh` session:
+
+```
+$ tctl bots add github-demo --roles=access --token=github-demo –logins=username
+```
+
+## Step 3/3. Configuring GitHub Actions
+
+Now that the bot has been successfully created, github actions can succesfully function as part of the Teleport cluster.
```

Comment:
```suggestion
Now that the bot has been successfully created, GitHub Actions can successfully function as part of the Teleport cluster.
```

Reply:
P.S. the use of "successfully" in succession (say that three times fast) here could sound redundant. Consider removing one. I'd suggest the first instance.